wd_helper.au3

```diff
@@ -1150,7 +1150,7 @@ EndFunc   ;==>_WD_GetShadowRoot
 Func _WD_SelectFiles($sSession, $sStrategy, $sSelector, $sFilename)
 	Local Const $sFuncName = "_WD_SelectFiles"
 
-	Local $sResult, $sJsonElement, $sSavedEscape
+	Local $sResult = "0", $sJsonElement, $sSavedEscape
```

Comment:
Same issue here.

Reply:
no here is not the same issue becuase when `_WD_FindElement()` return Error then `$sResult` will not be set to `"0"`

```AutoIt
	Local $sElement = _WD_FindElement($sSession, $sStrategy, $sSelector)
	Local $iErr = @error

	If $iErr = $_WD_ERROR_Success Then

```

We can do this by moving this part:
```AutoIt
		If $iErr = $_WD_ERROR_Success Then
			$sJsonElement = '{"' & $_WD_ELEMENT_ID & '":"' & $sElement & '"}'
			$sResult = _WD_ExecuteScript($sSession, "return arguments[0].files.length", $sJsonElement, Default, $_WD_JSON_Value)
			$iErr = @error
			If @error Then $sResult = "0"
		Else
			$sResult = "0"
		EndIf

```

out of the main `If....EndIf` area

Reply:
You are correct. In this case, I would stick with your original change and remove these lines --

```
		Else
			$sResult = "0"
```